### ISSUE TITLE
Unable to search and update NVDA automatically despite set to do so

### ISSUE THREAD

**Issue by PratikP1** [nvaccess/nvda#15040]
### Description of the problem
Since approximately after the release of NVDA 2023-1, My NVDA install has not been automatically updating the alpha builds even though the setting has been set to check for automatic updates. I have been manually checking for and installing updates through the  help menu. I cannot be certain of the exact timing for the beginning of this problem as I was not watching for this issue. See attempted resolution section below.

### Steps to reproduce:
1. Set NVDA to update automatically through preferences/settings/general/ "automatically check updates to NVDA".
2. In the same dialog box, check the box "notify for pending update on startup".
3. Wait for NVDA update to become available. As I have been testing with alpha builds, it should take a few days for an update to become available.
4. Go to NVDA's menu/help/check for update if an update does not appear automatically.
5. Update NVDA if a new build is available.

### Actual behavior:
NVDA does not check for updates automatically.

### Expected behavior:
NVDA should check for updates and present a dialog box to permit the operator to install the update.

### NVDA logs, crash dumps and other attachments:
[NVDA Update Log.txt](https://github.com/nvaccess/nvda/files/11805515/NVDA.Update.Log.txt)

#### NVDA installed/portable/running from source:
Installed on machine

#### NVDA version:
alpha-28462,3d78d701

#### Windows version:
Windows 11 23H2 (Build 25393.1)

#### Name and version of other software in use when reproducing the issue:
NA

#### Other information about your system:
NA

#### Does the issue still occur after restarting your computer?
yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
I have tried NVDA 2023-1 and the same issue occurs

#### If NVDA add-ons are disabled, is your problem still occurring?
yes. I have attempted to keep NVDA running without add-ons for a week while Alpha builds were released.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes.

### Attempted solutions
1. I have attempted to install other versions of NVDA including the released version as well as beta.
2. I have checked network settings to ensure that the OS is not blocking NVDA. However it should be noted that manual updates via the help menu works without any difficulty.
3. As a last resort, I completely uninstalled NVDA, deleted all user configuration files, restarted Windows, deleted remaining files in the program files directory, and reinstalled NVDA after downloading the latest snapshot.
4. Rather than copy over any configuration files, I manually installed each add-on and completely reconfigured NVDA manually.
5. Automatic updates are still not working. I see that there is another alpha snapshot available but NVDA is not automatically checking for updates.

**Comment by Adriani90** [comment]
@PratikP1 are you still having this issue with last NVDA alphas?

**Comment by Adriani90** [comment]
No updates from the initial author upon request since more than 1 year, closing as abandoned. If you are still having this issue, please comment and we can reopen.
